docs/learn/fundamentals/transactions/unique-source-for-operation.mdx

```diff
@@ -0,0 +1,112 @@
+---
+title: "When Operation Source Accounts differ from that of the Parent Transaction"
+sidebar_label: Unique Source Accounts for Operations
+description: "Learn"
+sidebar_position: 40
+---
+
+# When Operation Source Accounts differ from that of the Parent Transaction
```

Comment:
on the markdown side of things, having this "top-level" single-`#` heading is redundant if it's the same as the title.